src/core/login.ts

```diff
@@ -35,6 +35,11 @@ export class Login {
     });
   }
 
+  async userCookie(JSESSIONID: string, li_at: string): Promise<Client> {
+    this.setRequestHeaders({ cookies: { JSESSIONID, li_at } });
+    return this.client;
```

Comment:
Please save the cookie to the sessions file for future caching. (See this functionality in the userPass function)